tests/stdlib/tordsets.nim

```diff
@@ -0,0 +1,97 @@
+import std/ordsets
```

Comment:
you've changed this to `import std/ordsets` (good) but the problem remains:
`import ordsets` should not compile IMO, there's a policy (i forgot where) that new modules show require std prefix; that's to avoid clashes with nimble packages

Reply:
I've pushed the move to lib/std, I'm switching off now, but I can check with him tomorrow. Thank you for the review, this is literally my first nim code so I'm learning along.

Reply:
I've checked RFCs about nimble and naming and other that the std/ prefix as you suggested I couldn't find any clear criteria about whether it should go on folder lib/std or lib/pure/collections. What's your executive decision on this?
I've amended the commits to be clearer, 1. Full change over intset file for an easier diff of the change. 2. Create ordsets and intsets exports it. 3. Just the tentative file move to to /lib/std

Reply:
see https://github.com/nim-lang/Nim/pull/8024#issuecomment-396943421
> There is no reason whatsoever we can't have a long deprecation period for import strutils and the like, we can even decide the old stdlib modules should be seen as "keyword-like" and do not have to deprecate it at all (!) but all the new stuff ends up in std.

I'm adding this convention in this PR: https://github.com/nim-lang/Nim/pull/15590

Reply:
The last commit moves `ordsets.nim` to `lib/std`. Is there anything that needs doing?